### DISCUSSION TITLE
viewing output from a script run on a trigger

### DISCUSSION THREAD

**Discussion by bbc5008** [sadieblue/agilityblue.workspace-scripts#37] [Q&A] [answered]
Hello All,
Just curious, but is there a way to save/view output from a script run on a trigger?  i'm looking into this for debugging purposes.

**Comment by mrabb24** [comment, marked as answer]
Hello there,

You can view/export the results of any script output by navigating to the script details page and clicking on the script execution results tab on the left (should be the last tab). The additional information field contains the output of a script.

![image](https://github.com/user-attachments/assets/ccf7e8aa-5d21-417a-ad7c-59c0c3959ad1)

Matt
